README.md

```diff
@@ -156,44 +167,23 @@ point to the `master` branch of this repo.
 polars = { git = "https://github.com/pola-rs/polars", rev = "<optional git tag>" }
 ```
 
-#### Rust version
-
 Required Rust version `>=1.58`
 
-## Documentation
+### Node.js
```

Comment:
I think we can remove this from this README. The nodejs bindings have their own dedicated repo.

Reply:
Removed